docs/api/alerting/list_rule_types.asciidoc

```diff
@@ -4,7 +4,11 @@
 <titleabbrev>List rule types</titleabbrev>
 ++++
 
-Retrieve a list of alerting rule types.
+Retrieve a list of alerting rule types that the user is authorized to access.
+
+Each rule type includes a list of consumer features. Within these features, users are authorized to perform either `read` or `all` operations on rules of that type. This helps determine which rule types users can read, but not create or modify.
+
+NOTE: Creating some rule types is limited to specific features.  These rule types are not available when <<defining-alerts, defining rules>> in <<management,Stack Management>>.
```

Comment:
**nit**: to avoid confusion about being able to create rule types.
```suggestion
NOTE: Some rule types are limited to specific features.  These rule types are not available when <<defining-alerts, defining rules>> in <<management,Stack Management>>.
```